ansible/roles/openshift_node_config/tasks/main.yml

```diff
@@ -0,0 +1,10 @@
+---
+# tasks file for openshift_node_config
+- name: Edit OpenShift node config
+  lineinfile:
+    dest: /etc/sysconfig/atomic-openshift-node
+    line: "{{item.key }}={{ item.value }}"
```

Comment:
@joelddiaz New regex is tested and working as expected but since the upgrade now throw a deprecation warning. 'Using bare variables is deprecated'.